lib/omniship/package.rb

```diff
@@ -117,7 +106,46 @@ def attribute_from_metric_or_imperial(obj, klass, metric_unit, imperial_unit)
         return klass.new(obj, (@unit_system == :imperial ? imperial_unit : metric_unit))
       end
     end
-    
+
+    def determine_dimensions(values)
+      values.reject!(&:nil?)
```

Comment:
or you could use `values.reject!(&:blank?)` which will remove nil, "", "\n", " ", "\n\r"